scripts/spectrum-vars.js

```diff
@@ -171,9 +171,13 @@ const foundVars = await findUsedVars();
 spectrumPaths.forEach((spectrumPath, i) => {
     const packageDir = ['styles'];
     const isExpress = i === 1;
-    if (isExpress) packageDir.push('express');
+    if (isExpress) {
+        packageDir.push('express');
+    }
     themes.forEach((theme) => {
-        if (isExpress && ['lightest', 'darkest'].includes(theme)) return;
+        if (isExpress && ['lightest', 'darkest'].includes(theme)) {
```

Comment:
can we discard `lightest` and `darkest` from here too?

Reply:
I would prefer not to change this file, as it was recently reviewed during the S2 "plumbing" work, and we decided to maintain the same structure as Express.